File: java/ReverseSentence.java
public class ReverseSentence {
    public static void main(String[] args) {
        System.out.println(reverseString("Hello World"));
    }

    public static String reverseString(String sentence){
        String [] strArr = sentence.split(" ");
        StringBuilder resStr = new StringBuilder("");
        for(int i = strArr.length -1; i>=0;i--){
            resStr.append(strArr[i]);
        }

        return resStr.toString();
    }
}

File: js/reverseSentence.js
function resverseSentence(sentence){
    var resStr="";
    var sentenceArr = sentence.split(" ");

    for(let i =sentenceArr.length-1;i>=0;i--){
        resStr = resStr + sentenceArr[i];
    }

    return resStr;
}

console.log(resverseSentence("Hello World"));